lib/msf/core/exploit/remote/kerberos/ticket.rb

```diff
@@ -46,7 +46,13 @@ def create_ticket(enc_key:, start_time:, end_time:, sname:, flags:, domain:, use
             )
             if save_ccache
               ccache = generate_ccache(ticket, opts: opts)
-              path = store_loot('kerberos_ticket.ccache', 'application/octet-stream', domain, ccache.encode)
+
+              info = []
+              info << "realm: #{opts[:realm].upcase}"
+              info << "serviceName: #{opts[:server].to_s.downcase}"
+              info << "username: #{opts[:client].to_s.downcase}"
+
+              path = store_loot('mit.kerberos.ccache', 'application/octet-stream', domain, ccache.encode, nil, info.join(', '))
```

Comment:
This change makes it so the forged tickets are compatible with the current caching mechanism.